core/services/llo/onchain_channel_definition_cache.go

```diff
@@ -121,6 +126,20 @@ func filterName(addr common.Address, donID uint32) string {
 
 func NewChannelDefinitionCache(lggr logger.Logger, orm ChannelDefinitionCacheORM, client HTTPClient, lp logpoller.LogPoller, addr common.Address, donID uint32, fromBlock int64, options ...Option) llotypes.ChannelDefinitionCache {
 	filterName := logpoller.FilterName("OCR3 LLO ChannelDefinitionCachePoller", addr.String(), donID)
+	donIDTopic := common.BigToHash(big.NewInt(int64(donID)))
+
+	exprs := []query.Expression{
+		logpoller.NewAddressFilter(addr),
+		logpoller.NewEventSigFilter(NewChannelDefinition),
+		logpoller.NewEventByTopicFilter(1, []logpoller.HashedValueComparator{
+			{Value: donIDTopic, Operator: primitives.Eq},
+		}),
+		// NOTE: Optimize for fast pickup of new channel definitions. On
+		// Arbitrum, finalization can take tens of minutes
+		// (https://grafana.ops.prod.cldev.sh/d/e0453cc9-4b4a-41e1-9f01-7c21de805b39/blockchain-finality-and-gas?orgId=1&var-env=All&var-network_name=ethereum-testnet-sepolia-arbitrum-1&var-network_name=ethereum-mainnet-arbitrum-1&from=1732460992641&to=1732547392641)
+		query.Confidence(primitives.Unconfirmed),
```

Comment:
what are the potential risks we are accepting here for this optimization?

Reply:
None really, we have always done it this way